Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM gliderlabs/alpine:3.4
+FROM gliderlabs/alpine:3.8
 MAINTAINER Hypothes.is Project and contributors
 
 # Install system build and runtime dependencies.
-RUN apk-install ca-certificates collectd curl nodejs python3 supervisor
+RUN apk-install ca-certificates collectd curl nodejs nodejs-npm python3 supervisor
```

Comment:
Alpine's `nodejs` package no longer installs `npm`, you need `nodejs-npm` as well